ar_exercise/exercise.rb

```diff
@@ -34,14 +34,31 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+
 # get all tenants older than 65
+SELECT * FROM tenants WHERE age >= 65;
 # get all apartments whose price is greater than $2300
+SELECT * FROM apartments WHERE monthly_rent >= 2300;
 # get the apartment with the address "6005 Damien Corners"
+SELECT * FROM apartments WHERE address = '6005 Damien Corners';
 # get all tenants in that apartment
+SELECT * FROM tenants WHERE apartment_id = 6;
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+
+idName = SELECT name, tenant_id FROM tenants;
```

Comment:
Combining SQL and Ruby here won't work. Ideally you would use Active Record commands to retrieve what you need from the database.